.travis.yml

```diff
@@ -63,5 +65,17 @@ install:
 script:
   - npm run compile
   - npm run lint
-  - npm run test
-  - codecov
\ No newline at end of file
+  - |
+    if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+      npm run test
+    fi
+  - |
+    if [[ $TRAVIS_OS_NAME == "osx" ]]; then
+      # Due to slowness of OS X on Travis
+      # https://github.com/travis-ci/travis-ci/issues/6095
```

Comment:
Here is where I've disabled the tests on the OS X version of Travis because of known slowness. We will still be able to run the tests locally. And, we have Windows and Linux on the CI for sanity purposes.